pkg/google/gke/pricing_map.go

```diff
@@ -265,14 +272,14 @@ func (pm *PricingMap) Populate(ctx context.Context, billingService *billingv1.Cl
 						break
 					}
 				}
-				if strings.Contains(data.Description, "Confidential") {
-					log.Printf("Storage class contains Confidential: %s\n%s\n", storageClass, data.Description)
-					continue
-				}
 				if storageClass == "" {
 					log.Printf("Storage class not found for %s. Skipping", data.Description)
 					continue
 				}
+				if strings.Contains(data.Description, "Confidential") {
+					log.Printf("Storage class contains Confidential: %s\n%s\n", storageClass, data.Description)
+					continue
+				}
```

Comment:
I'm moving this after the first `storageClass` check since that one should take priority.